src/useRootClose.js

```diff
@@ -69,23 +71,25 @@ function useRootClose(
   useEffect(() => {
     if (disabled || ref == null) return undefined;
 
+    const doc = ownerDocument(ref && ref.current);
```

Comment:
```suggestion
    const doc = ownerDocument(ref.current);
```

we already check that ref is defined above